web/themes/new_weather_theme/templates/layout/page--point.html.twig

```diff
@@ -38,76 +38,76 @@
+    </div>
     {% else %}
-      <wx-tabbed-nav class="display-block position-relative">
-        <div id="tablist-label" class="usa-sr-only">{{ "Weather information sections" | t }}</div>
-        <div class="position-sticky top-0 z-500 bg-white border-base-light border-top-0 border-bottom-2px margin-y-0">
-          <div class="grid-container padding-x-0">
+    <wx-tabbed-nav class="display-block position-relative">
+      <div id="tablist-label" class="usa-sr-only">{{ "Weather information sections" | t }}</div>
+      <div class="position-sticky top-0 z-500 bg-white border-base-light border-top-0 border-bottom-2px margin-y-0">
+        <div class="grid-container padding-x-0">
           <div role="tablist" class="tab-buttons display-flex flex-row top-4 grid-col-12" aria-labelledby="tablist-label">
             {% if point.alerts.items | length > 0 %}
-              <button role="tab" id="alerts-tab-button" class="tab-button" data-tab-name="alerts" aria-controls="alerts">{{ "Alerts" | t }}</button>
+            <button role="tab" id="alerts-tab-button" class="tab-button" data-tab-name="alerts" aria-controls="alerts">{{ "Alerts" | t }}</button>
             {% endif %}
-              <button role="tab" id="current-conditions-tab-button" class="tab-button" data-tab-name="today" aria-controls="today">{{ "Today" | t }}</button>
-              <button role="tab" id="daily-tab-button" class="tab-button" data-tab-name="daily" aria-controls="daily">{{ "7 Days" | t }}</button>
-            </div>
+            <button role="tab" id="current-conditions-tab-button" class="tab-button" data-tab-name="today" aria-controls="today">{{ "Today" | t }}</button>
+            <button role="tab" id="daily-tab-button" class="tab-button" data-tab-name="daily" aria-controls="daily">{{ "7 Days" | t }}</button>
           </div>
         </div>
+      </div>
+
+      <div class="bg-base-lightest padding-top-3 padding-bottom-8">
+        {% if point.alerts.items | length > 0 %}
+        <div class="wx-tab-container wx-focus-offset-205" id="alerts" role="tabpanel" aria-labelledby="alerts-tab-button" tabindex="0">
+          {% include(directory ~ "/templates/point/alerts.html.twig") with { alerts: point.alerts.items } %}
+        </div>
+        {% endif %}
 
-        <div class="bg-base-lightest padding-top-3 padding-bottom-8">
-          {% if point.alerts.items | length > 0 %}
-          <div class="wx-tab-container wx-focus-offset-205" id="alerts" role="tabpanel" aria-labelledby="alerts-tab-button" tabindex="0">
-            {% include(directory ~ "/templates/point/alerts.html.twig") with { alerts: point.alerts.items } %}
+        <div class="wx-tab-container wx-focus-offset-205" id="today" role="tabpanel" aria-labelledby="current-conditions-tab-button" tabindex="0">
+          <div class="grid-container tablet:padding-x-2 desktop:padding-x-4 padding-x-0">
+            <div class="grid-row grid-gap-lg">
```

Comment:
Did we undo our replacement of the `grid-gap-lg` with our custom utility here and elsewhere? I am seeing the issue show up again
![Screenshot of mobile scrolling](https://github.com/user-attachments/assets/e97d1a6f-fe2b-4284-a64c-31a5645f288f)